content/docs/command-reference/add.md

```diff
@@ -37,29 +37,29 @@ Under the hood, a few actions are taken for each file (or directory) in
    for more details.)
 3. Attempt to replace the file with a link to the cached data (more details
    further down).
-4. Create a corresponding DVC-file to store the file (as an
+4. Create a corresponding `.dvc` file to store the file (as an
    <abbr>output</abbr>), using its path and hash to identify the cached data.
-   Unless the `-f` option is used, the DVC-file name generated by default is
+   Unless the `-f` option is used, the `.dvc` file name generated by default is
    `<file>.dvc`, where `<file>` is the file name of the first target.
 5. Unless `dvc init --no-scm` was used when initializing the project, add the
    `targets` to `.gitignore` in order to prevent them from being committed to
    the Git repository.
 6. Instructions are printed showing `git` commands for adding the files, if
    appropriate.
 
-Summarizing, the result is that the target data is replaced small DVC-files can
-be tracked with Git. See [DVC-File Format](/doc/user-guide/dvc-file-format) for
-more details.
+Summarizing, the result is that the target data is replaced small `.dvc` files
+can be tracked with Git. See [DVC-File Format](/doc/user-guide/dvc-file-format)
```

Comment:
Please add missing words when you see any:
- "replaced **by**"
- "**that** can be tracked"